framework/models.py

```diff
@@ -0,0 +1,69 @@
+# External Libraries
+from pony.orm import Set, Database, Required, PrimaryKey, db_session, Optional
+
+db = Database()
+
+
+class Base:
+    @classmethod
+    @db_session
+    def exists(cls, id_: str):
+        return cls.get(id=id_) is not None  # pylint: disable=no-member
+
+    @classmethod
+    @db_session
+    def get_s(cls, arg):
+        return cls.get(id=arg)  # pylint: disable=no-member
+
+    @property
+    @db_session
+    def json(self):
+        return self.__class__[self.id].to_dict(with_collections=True, exclude="password")  # pylint: disable=no-member
+
+
+class Mod(db.Entity, Base):
+    id = PrimaryKey(str)
+    title = Required(str)
+    icon = Optional(str, nullable=True)
+    path = Required(str)
+    # TODO: Media
+    tagline = Required(str)
+    description = Required(str)
+    website = Required(str)
+    # TODO: Category
+    released_at = Required(int)
+    last_updated = Required(int)
+    downloads = Required(int)
+    authors = Set('User', reverse='mods')
+    favorite_by = Set('User', reverse='favorites')
+    reviews = Set('Review', reverse="mod")
+    verified = Required(bool)
+
```

Comment:
Add a field based on mod status (in development, playtesting, released) (gets fixed at release when `released_at` is set).

Reply:
Fixed in upcoming commit